layouts/partials/sidebar/modal-content.css

```diff
@@ -0,0 +1,122 @@
+    "font-size: {{ ($.Scratch.Get "settings").product_reviews_font_size }};" +
+    "font-family: {{ ($.Scratch.Get "settings").product_reviews_font_family }};" +
+    "font-weight: {{ ($.Scratch.Get "settings").product_reviews_font_weight }};" +
+    "margin-left: {{ ($.Scratch.Get "settings").product_reviews_margin_left }};" +
+    "top: {{ ($.Scratch.Get "settings").product_reviews_top }};" +
+"}" +
+
+".tvp-rate {" +
+    "width: {{ ($.Scratch.Get "settings").product_rate_width }};" +
+    "height: {{ ($.Scratch.Get "settings").product_rate_height }};" +
+    "background-image: url('{{ $.Site.BaseURL }}/sidebar/{{ ($.Scratch.Get "settings").product_rating_background_image_url }}');" +
+    "background-size: cover;" +
+"}" +
+
+".tvp-rate.full {" +
+    "background-position: 0 0;" +
+"}" +
+
+".tvp-rate.half {" +
+    "background-position: 0 -41px;" +
+"}" +
+
+".tvp-rate.empty {" +
+    "background-position: 0 -20px;" +
+"}" +
+
+".tvp-product-image {" +
+    "background-size: {{ ($.Scratch.Get "settings").product_thumbnail_background_image_size }};" +
+"}" +
+
+".tvp-product-image-overlay {" +
+    "background-color: {{ ($.Scratch.Get "settings").product_thumbnail_image_overlay_color }};" +
+"}" +
+
+".tvp-product-popup {" +
+    "width: {{ ($.Scratch.Get "settings").product_popup_width }};" +
+    "background: {{ ($.Scratch.Get "settings").product_popup_background }};" +
+    "padding: {{ ($.Scratch.Get "settings").product_popup_padding }};" +
+    "border: {{ ($.Scratch.Get "settings").product_popup_border }};" +
+    "box-shadow: {{ ($.Scratch.Get "settings").product_popup_box_shadow }};" +
+"}" +
+
+".tvp-product-popup-image {" +
+    "background-size: {{ ($.Scratch.Get "settings").product_thumbnail_background_image_size }};" +
+    "border: {{ ($.Scratch.Get "settings").product_popup_thumbnail_border }};" +
+"}" +
+
+".tvp-product-cta {" +
+    "background: {{ ($.Scratch.Get "settings").product_popup_cta_background }};" +
+    "border: {{ ($.Scratch.Get "settings").product_popup_cta_border }};" +
+    "border-bottom: {{ ($.Scratch.Get "settings").product_popup_cta_border_bottom }};" +
+    "color: {{ ($.Scratch.Get "settings").product_popup_cta_font_color }};" +
+    "text-transform: {{ ($.Scratch.Get "settings").product_popup_cta_text_transform }};" +
+    "font-size: {{ ($.Scratch.Get "settings").product_popup_cta_font_size }};" +
+    "font-family: {{ ($.Scratch.Get "settings").product_popup_cta_font_family }};" +
+    "font-weight: {{ ($.Scratch.Get "settings").product_popup_cta_font_weight }};" +
+    "height: {{ ($.Scratch.Get "settings").product_popup_cta_height }};" +
+"}" +
+
+".tvp-product-cta:hover {" +
+    "background: {{ ($.Scratch.Get "settings").product_popup_cta_background_hover }};" +
+"}" +
+
+".tvp-arrow-indicator:before {" +
+    "border-left: 13px solid {{ ($.Scratch.Get "settings").product_popup_arrow_indicator_border_color }};" +
+"}" +
+
+".ss-scroll {" +
```

Comment:
I think we shall have a class called `tvp-scrollbar` and have here all the skin/theme related CSS (colors, backgrounds, etc.)

Leaving in `vendor/simple-scrollbar.css` all the CSS required for the plugin to function, with their CSS classes `.ss-scroll`, etc.